tests/includes/aws_ami_creation.sh

```diff
@@ -1,15 +1,32 @@
 launch_and_wait_ec2() {
-	local instance_name instance_id_result
+	local instance_name instance_id_result instance_type
 
 	instance_name=${1}
 	instance_image_id=${2}
 	subnet_id=${3}
 	instance_id_result=${4}
 
+	arch="amd64"
+	if [[ -n ${MODEL_ARCH} ]]; then
+		arch="${MODEL_ARCH}"
+	fi
+	case "${arch}" in
+	"amd64")
+		instance_type="t2.medium"
+		;;
+	"arm64")
+		instance_type="c6g.large"
```

Comment:
c6 will work but an m instance is probably the better choice considering what we are doing.

Maybe try an `m7g.medium`